modules/social_features/social_profile/social_profile.install

```diff
@@ -202,3 +203,28 @@ function social_profile_update_121001(): void {
 function social_profile_update_130001(): void {
   user_role_grant_permissions('sitemanager', ['create terms in profile_tag']);
 }
+
+/**
+ * Update views /all-members to hide blocked members.
+ */
+function social_profile_update_130002(): void {
+  $moduleHandler = \Drupal::moduleHandler();
+  if ($moduleHandler->moduleExists('views') === FALSE ||
+    $moduleHandler->moduleExists('user') === FALSE ||
+    $moduleHandler->moduleExists('profile') === FALSE
+  ) {
+    return;
+  }
+
+  $config_file = \Drupal::service('extension.list.module')->getPath('social_profile') . '/config/install/views.view.newest_users.yml';
```

Comment:
We should also copy the config and put it in /config/update/social_profile_update_130002.yml.
Reason for this is that the config in /install might change over time and we want to be sure that when the update hook runs it has the config it was tested for at that particular moment in time.